cloud-haven-react-app/src/components/VendorAppPage/VendorAppPage.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
-import { CustomForm } from '../components';
-import { Container } from 'react-bootstrap';
+import { FileList, CustomForm } from '../components';
 import './VendorAppPage.css';
 
+const isOnVendorHomePage = false;
+
 export default props => {
     return (
-        <Container>
+        <React.Fragment>
```

Comment:
I'm not sure what a React.Fragment is? Why do we want to use it here instead of a Container?

Reply:
React bootstrap's `Container` adds margins to the page. Removing it allowed me to have the file list line up with the edge of the page frame. I could've put a regular `div` here, but I chose to use React.Fragment since adding a `div` without an id or class would have little purpose other than meeting that requirement of having one outermost element. A `div` without purpose would be unnecessarily adding layers to the DOM. A [react fragment](https://reactjs.org/docs/fragments.html) allows you to group the parts of the component together without adding extra elements to the DOM.